src/support/kuzzle.ts

```diff
@@ -41,24 +43,33 @@ export class KuzzleSDK {
 
   private password: string;
 
-  private loginTTL: string;
+  private refreshLogin: boolean;
+
+  private protocol: string;
+
+  private refreshTimer?: NodeJS.Timeout;
 
   constructor(options: any) {
     this.host = options.host
     this.port = parseInt(options.port, 10)
     this.ssl = options.ssl || this.port === 443
     this.username = options.username
     this.password = options.password
-    this.loginTTL = options.loginTTL || '10s'
+    this.refreshLogin = options.refreshLogin || false
```

Comment:
Why should this be an option?

As a user, I expect the CLI to take care of how the authentication token is created and refreshed.

If any command runs longer than the default login TTL (which is 90s), then I think that Kourou should automatically renew its token until the command ends.

Also, renewing every second or so seems a bit too often to me. If you have a login TTL of 90s, why not refreshing every 80s or so (10s to account for network latency seems largely enough)

What do you think?

Reply:
Yes you're right, I'm gonna refresh it all the time if needed :+1: